pkg/apis/resource/types.go

```diff
@@ -233,6 +233,9 @@ type ResourcePool struct {
 const ResourceSliceMaxSharedCapacity = 128
 const ResourceSliceMaxDevices = 128
 const PoolNameMaxLength = validation.DNS1123SubdomainMaxLength // Same as for a single node name.
+const BindingConditionsMaxSize = 4
+const BindingFailureConditionsMaxSize = 4
+const BindingTimeoutDefaultSeconds = 600 // Default timeout for binding conditions, in seconds.
```

Comment:
drop `BindingTimeoutDefaultSeconds` from the REST API packages since this is internal to the scheduler now (applies to all REST API types.go files, can move this local to where it is used in the scheduler for now)

Reply:
Thank you! I have update to drop `BindingTimeoutDefaultSeconds`.